src/machine.adoc

```diff
@@ -2508,14 +2508,16 @@ available only to machine-mode.
 
 ==== Main Memory versus I/O versus Vacant Regions
 
-The most important characterization of a given memory address range is
-whether it holds regular main memory, or I/O devices, or is vacant.
+The most important characterization of a specific memory address range is
+whether it holds regular main memory or I/O devices, or if it is vacant.
 Regular main memory is required to have a number of properties,
-specified below, whereas I/O devices can have a much broader range of
-attributes. Memory regions that do not fit into regular main memory, for
-example, device scratchpad RAMs, are categorized as I/O regions. Vacant
-regions are also classified as I/O regions but with attributes
-specifying that no accesses are supported.
+which are specified in the following sections. However, I/O device regions can have a much broader  
+range of attributes. Address regions that do not fit into regular main memory are categorized as 
+I/O; for example, regions that contain device control registers. Finally, there are 
+other address regions that are neither memory nor I/O. These regions are called vacant
+regions. Vacant regions include attributes that specify access to these regions isn't 
+supported. Vacant regions can also be classified as I/O regions, but are never accessible. Any 
```

Comment:
> Vacant regions include attributes that specify access to these regions isn't 
supported. Vacant regions can also be classified as I/O regions, but are never accessible.

This is a slight change from the previous, but is it really clarifying?  I had thought that {Main memory, I/O, Vacant} was a non-overlapping set.